Makefile

```diff
@@ -61,13 +61,19 @@ help:
 	@echo 'Set the RELATIVE variable to 1 to enable relative urls                    '
 	@echo '                                                                          '
 
-html:
+icalendar:
+	python -c "import icalendar" || pip install icalendar
```

Comment:
This part is not necessary. Also, it is not necessary to install icalendar every time HTML is run.

Reply:
What I mean by "this part," I am referring to the python -c "import icalendar" bit.

Reply:
it is not installing icalendar every time HTML is run.  learn you some Makefile before you comment.

Reply:
and it is necessary, unless you want to put icalendar as a dependency in the docs

Reply:
trust me - this code does exactly what it is intended to do.
We are still missing a dependency.  That's what I am asking you to resolve.

Reply:
commenting on code you haven't run and don't understand is not helpful.  It's just a waste of everyone's time.  Unless the comment is "does this do x?"